x-pack/plugins/security/public/account_management/user_profile/user_profile_api_client.ts

```diff
@@ -76,6 +113,28 @@ export class UserProfileAPIClient {
     });
   }
 
+  /**
+   * Suggests multiple user profiles by search criteria.
+   *
+   * Note: This endpoint is not provided out-of-the-box by the platform. You need to expose your own
+   * version within your app. An example of how to do this can be found in:
+   * `examples/user_profile_examples/server/plugin.ts`
+   *
+   * @param path Path to your app's suggest endpoint.
+   * @param params Suggest operation parameters.
+   * @param params.name Query string used to match name-related fields in user profiles. The
+   * following fields are treated as name-related: username, full_name and email.
+   * @param params.size Desired number of suggestion to return. The default value is 10.
+   * @param params.dataPath By default, suggest API returns user information, but does not return
+   * any user data. The optional "dataPath" parameter can be used to return personal data for this
+   * user (within `kibana` namespace only).
+   */
```

Comment:
The `suggest` endpoint is a bit of a strange one. We can't expose it out-of-the-box within the platform since its privilege checks need to be hard-coded server-side. This means that each app has to expose their own endpoint. 

However, I still wanted to provide some functionality and the correct typings as part of the User Profile API Client to improve discoverability, hence this method requiring the `path` to the app's own suggest endpoint.